cluster/manifests/vertical-pod-autoscaler/admission-controller-deployment.yaml

```diff
@@ -0,0 +1,47 @@
+{{ if eq .ConfigItems.vpa_enabled "true" }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: vpa-admission-controller
+  namespace: kube-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        application: vpa-admission-controller
+    spec:
+      serviceAccountName: system
+      containers:
+      - name: admission-controller
+        image: pierone.stups.zalan.do/teapot/vpa-admission-controller:test1
```

Comment:
flags should be specified here instead of in the docker image, see https://github.bus.zalan.do/teapot/vertical-pod-autoscaler/commit/366b393d5d96076bf1fb0f7af53386cd3a1d35fb